common.py

```diff
@@ -45,7 +48,32 @@ def governanceProposal(deployment):
                 deploymentInfo['name'],
                 {'from': GOV_MULTISIG}
             )
+            proposalId = propose_tx.events['ProposalCreated'][0][0]['proposalId']
+
+            # Move forward 30s so that we can vote
+            timetravel(30)
+
+            # Vote on Proposal
+            print("Voting on the proposal")
+            ogvGovernor.castVote(proposalId, 1, {'from': GOV_MULTISIG})
+            
+            # 3 days to queue (+2 for buffer since contract has a different BLOCKS_PER_DAY now)
+            timetravel(86400 * 5)
+
+            print("Queueing proposal")
+            ogvGovernor.queue(proposalId, {'from': GOV_MULTISIG})
+
+            # 2 day timelock (+1 day for buffer)
+            timetravel(86400 * 3)
+
+            print("Executing proposal")
+            ogvGovernor.execute(proposalId, {'from': GOV_MULTISIG})
+
+
         print("Execute the following transaction to create OGV Governance proposal")
         print("To: {}".format(propose_tx.receiver))
         print("Data: {}".format(propose_tx.input))
 
+def timetravel(seconds):
```

Comment:
nit: maybe just add a comment that this moves the chain up for a certain number of blocks that correspond to voting periods in the contracts, but doesn't actually increase the block timestamps (there is an `evm_increaseTime` RPC for that)

Reply:
`brownie.chain.sleep` is actually used in this function to increase the block timestamp as well. [Here's a link](https://eth-brownie.readthedocs.io/en/stable/core-chain.html#time-travel) to the brownie docs

Reply:
ah right, thanks 👍